package.json

```diff
@@ -0,0 +1,330 @@
+    "process": "^0.11.10",
+    "pkg-dir": "^3.0.0",
+    "posix-character-classes": "^0.1.1",
+    "promise-inflight": "^1.0.1",
+    "process-nextick-args": "^2.0.1",
+    "public-encrypt": "^4.0.3",
+    "pump": "^3.0.0",
+    "prr": "^1.0.1",
+    "pumpify": "^1.5.1",
+    "punycode": "^1.4.1",
+    "querystring": "^0.2.0",
+    "querystring-es3": "^0.2.1",
+    "readable-stream": "^2.3.7",
+    "randombytes": "^2.1.0",
+    "randomfill": "^1.0.4",
+    "regex-not": "^1.0.2",
+    "readdirp": "^2.2.1",
+    "repeat-element": "^1.1.3",
+    "repeat-string": "^1.6.1",
+    "remove-trailing-separator": "^1.1.0",
+    "require-main-filename": "^2.0.0",
+    "resolve-dir": "^1.0.1",
+    "require-directory": "^2.1.1",
+    "resolve-from": "^3.0.0",
+    "resolve-url": "^0.2.1",
+    "resolve-cwd": "^2.0.0",
+    "ret": "^0.1.15",
+    "ripemd160": "^2.0.2",
+    "rimraf": "^2.7.1",
+    "run-queue": "^1.0.3",
+    "safe-buffer": "^5.2.0",
+    "schema-utils": "^1.0.0",
+    "safe-regex": "^1.1.0",
+    "semver": "^5.7.1",
+    "set-blocking": "^2.0.0",
+    "serialize-javascript": "^2.1.2",
+    "set-value": "^2.0.1",
+    "sha.js": "^2.4.11",
+    "setimmediate": "^1.0.5",
+    "shebang-command": "^1.2.0",
+    "signal-exit": "^3.0.2",
+    "shebang-regex": "^1.0.0",
+    "snapdragon": "^0.8.2",
+    "snapdragon-util": "^3.0.1",
+    "snapdragon-node": "^2.1.1",
+    "source-list-map": "^2.0.1",
+    "source-map-resolve": "^0.5.3",
+    "source-map": "^0.6.1",
+    "source-map-support": "^0.5.16",
+    "source-map-url": "^0.4.0",
+    "ssri": "^6.0.1",
+    "split-string": "^3.1.0",
+    "static-extend": "^0.1.2",
+    "stream-browserify": "^2.0.2",
+    "stream-each": "^1.2.3",
+    "stream-http": "^2.8.3",
+    "stream-shift": "^1.0.1",
+    "string-width": "^3.1.0",
+    "string_decoder": "^1.3.0",
+    "strip-ansi": "^5.2.0",
+    "supports-color": "^6.1.0",
+    "strip-eof": "^1.0.0",
+    "tapable": "^1.1.3",
+    "terser": "^4.6.6",
+    "through2": "^2.0.5",
+    "terser-webpack-plugin": "^1.4.3",
+    "timers-browserify": "^2.0.11",
+    "to-arraybuffer": "^1.0.1",
+    "to-regex": "^3.0.2",
+    "to-object-path": "^0.3.0",
+    "to-regex-range": "^2.1.1",
+    "tslib": "^1.11.1",
+    "tty-browserify": "^0.0.0",
+    "typedarray": "^0.0.6",
+    "union-value": "^1.0.1",
+    "unique-filename": "^1.1.1",
+    "unique-slug": "^2.0.2",
+    "unset-value": "^1.0.0",
+    "uri-js": "^4.2.2",
+    "upath": "^1.2.0",
+    "urix": "^0.1.0",
+    "url": "^0.11.0",
+    "use": "^3.1.1",
+    "util": "^0.11.1",
+    "util-deprecate": "^1.0.2",
+    "v8-compile-cache": "^2.0.3",
+    "vm-browserify": "^1.1.2",
+    "watchpack": "^1.6.0",
+    "webpack-sources": "^1.4.3",
+    "which": "^1.3.1",
+    "which-module": "^2.0.0",
+    "worker-farm": "^1.7.0",
+    "wrap-ansi": "^5.1.0",
+    "wrappy": "^1.0.2",
+    "xtend": "^4.0.2",
+    "y18n": "^4.0.0",
+    "yallist": "^3.1.1",
+    "yargs": "^13.2.4",
+    "yargs-parser": "^13.1.2"
```

Comment:
😲 Where is all of this coming from? Are you really using all of these dependencies?

Reply:
Deleted!!! Thanks 👍